.github/workflows/build_main_documentation.yml

```
@@ -76,7 +86,11 @@ jobs:
           pip install .
           cd ..
 
-          if [[ "${{ inputs.additional_args }}" != *"--not_python_module"* ]];
+          if [ "${{ inputs.package_path }}" ]
```

Don't we need a `-n` here?
```suggestion
          if [[ -n "${{ inputs.package_path }}" ]]
```